storage/src/database/backend.rs

```diff
@@ -0,0 +1,38 @@
+use crate::database::error::Error;
+use crate::Direction;
+#[derive(Clone)]
+pub enum BackendIteratorMode {
+    Start,
+    End,
+    From(Vec<u8>, Direction),
+}
+
+pub trait TezedgeDatabaseBackendStore {
+    fn put(&self, column: &'static str, key: Vec<u8>, value: Vec<u8>) -> Result<(), Error>;
```

Comment:
@mambisi @tizoc 
I would rather introduce here some alias types maybe, what do you thing? something like:
```
pub type Key = Vec<u8>;
pub type Value = Vec<u8>;

...
fn put(&self, column: &'static str, key: Key, value: Value) -> Result<(), Error>;

```